src/app/profile/overview/work-items/work-items.component.ts

```diff
@@ -37,7 +37,7 @@ export class WorkItemsComponent implements OnDestroy, OnInit  {
     this.subscriptions.push(userService.loggedInUser.subscribe(user => {
       this.loggedInUser = user;
       if (user.attributes) {
-        this.subscriptions.push(spaceService.getSpacesByUser(user.attributes.username, 10).subscribe(spaces => {
+        this.subscriptions.push(spaceService.getSpacesByUser(user.attributes.username).subscribe(spaces => {
```

Comment:
This limitation was intentional. I don't recall the exact reason, but there could be quite a lot of spaces shown here. We use recent spaces on the home dashboard for this very reason.

Perhaps @mindreeper2420 or @catrobson can best answer this question?

Reply:
The limit of 10 spaces was set so that there would be no scrolling within the card. On smaller screens, it has the possibility of introducing double scrollbars (scroll page content and scroll card content), so the cards are fixed at a height to display that desired number of items.

Reply:
@mindreeper2420 Hm yeah I guess that makes sense, but isn't it dependant on the system as well? For example, I logged in on my phone and the combobox is nicely scrollable; it'd easily be able to accommodate all my spaces. 

![android](https://user-images.githubusercontent.com/10425301/40789023-72e61a1a-64bf-11e8-8a9f-fe2aaf95599d.png)

Or would there be a way to fix the height of the combobox and make it scrollable instead while still allowing for all spaces to be shown? I feel like otherwise we're not delivering the information we're advertising.